_i18n/fr.yml

```diff
@@ -38,22 +62,24 @@ contact_page:
       usajobs: USAJOBS
       veterans_affairs: Veterans Affairs
     errors:
-      captcha_required: You must complete the spam prevention challenge. # TODO: Translate
+      captcha_required: Vous devez relever le défi de la prévention du spam. # TODO: Translate
```

Comment:
Remove the TODO?

```suggestion
      captcha_required: Vous devez relever le défi de la prévention du spam.
```

Reply:
Updated texts and removed TODO in 729b3fe.